data-structures-queues/take-front.js

```diff
@@ -0,0 +1,6 @@
+export default function takeFront(queue) {
+  if (queue.peek() === undefined) {
```

Comment:
This test is also not necessary since `queue.dequeue()` returns undefined if the queue is empty.